Gopkg.toml

```diff
@@ -7,6 +7,10 @@
   branch = "master"
   name = "github.com/GeertJohan/go.rice"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/dop251/goja"
+
```

Comment:
Weird that goja wasn't already here o.O

Reply:
Hm ... I could've probably dropped this ... I needed it to specify a specific branch (es6) but now it's no longer required - but I will leave it as it's going to get dropped either way when we move to go modules ;)

Reply:
This should probably be pinned to a specific `revision`, otherwise `dep ensure` would upgrade it I think. But yeah, moot point because of the Go modules switch.